src/XliffTasks/XliffTasks.csproj

```diff
@@ -26,7 +26,7 @@
     <InternalsVisibleTo Include="XliffTasks.Tests" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
     <PackageReference Include="Microsoft.Build.Framework" Version="15.1.1012" PrivateAssets="All" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.1.1012" PrivateAssets="All" />
     <PackageReference Include="Microsoft.NETCore.App" Version="2.0.3" PrivateAssets="All" />
```

Comment:
Is all this still needed? `Microsoft.NETCore.App.2.0.3` is marked "[deprecated](https://www.nuget.org/packages/Microsoft.NETCore.App/2.0.3)" so it should at least get an upgrade here.

Reply:
Upgraded the package.